src/locales/de.ts

```diff
@@ -21,7 +21,7 @@ export default {
     'storage.renameMessage': 'Name des Speicherorts ändern',
     'storage.remove': 'Speicherort entfernen',
     'storage.removeMessage': 'Verknüpfung des Speicherorts wird entfernt',
```

Comment:
`storage.removeMessage` is also changed.

The `enUS` language has this:

```json
'storage.removeMessage': 'Warning! This will permanently remove all notes locally stored on this location.',
```

Reply:
Thanks! Updated it.